src/SpamKiller.ts

```diff
@@ -155,7 +155,13 @@ export default class SpamKiller {
         if (this.sharedSettings.spam.allowedUrls.findIndex(u => hostname.endsWith(u) &&
         (hostname.replace(u, "").endsWith(".") || hostname.replace(u, "").length === 0)) !== -1) // Only allow matching base domain (zero length after replace) and subdomains (ends with ".")
             return;
-        this.addViolatingMessage(message, `Hey, ${message.author}, we require users to verify that they are human before they are allowed to post a link. If you are a human, react with :+1: to this message to gain link privileges. If you are a bot, please go spam somewhere else. 👍`);
+
+        const embed = new Discord.MessageEmbed()
+            .setTitle("Robot Check")
+            .setColor(0xffff00)
```

Comment:
I think this plain yellow looks a bit retro.

Maybe `#FFCC00` would be a better fit.
https://coolors.co/ffff00-ffcc00

Reply:
The png also looks a bit old.

Reply:
I normally pick something out of the Nuvola set of icons, but I've updated the color and picture.